CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# SV Contributing Guide
+
+## Workflow
+
+We follow the standard fork-based workflow:
+
+1. **Fork** this repository to your GitHub account.
+2. **Clone** your fork locally.
+3. **Create a new branch** for your change:  
+   `git checkout -b your-feature-name`
+4. **Commit and push** your changes to your branch.
+5. **Open a pull request** from your branch to the `main` branch of this repository.
+
+Please keep your pull requests focused to feature or issue. Focused smaller changes are easier to review and faster to merge.
+
+## Preparing
+This is a monorepo, meaning the repo holds multiple packages. It requires the use of [pnpm](https://pnpm.io/). You can [install pnpm](https://pnpm.io/installation) with:
+
+```bash
+npm i -g pnpm
+```
+
+For running certain packages and tests locally you will need to install [docker](https://docs.docker.com/get-started/get-docker).
+Linux users, you will have to ensure 'sudo' is not required. See [docker post install](https://docs.docker.com/engine/install/linux-postinstall/)
+
+`pnpm` commands run in the project's root directory will run on all sub-projects. You can checkout the code and install the dependencies with:
+
+```bash
+git clone git@github.com:YOUR_GITHUB_USER_HERE/cli.git
+cd cli
```

Comment:
I don't think its necessary to showcase how to clone a repo, if thats not clear there are probably better places to start.